deps/llvm.mk

```diff
@@ -403,20 +399,20 @@ endif # LLVM_GIT_VER_POLLY
 endif # USE_POLLY
 endif # LLVM_VER
 	# touch some extra files to ensure bisect works pretty well
+	touch -c $(LLVM_SRC_DIR).extracted
 	touch -c $(LLVM_SRC_DIR)/configure
 	touch -c $(LLVM_SRC_DIR)/CMakeLists.txt
 	echo 1 > $@
 
 # Apply version-specific LLVM patches
-LLVM_PATCH_PREV:=
-LLVM_PATCH_LIST:=
+LLVM_PATCH_PREV :=
 define LLVM_PATCH
-$$(LLVM_SRC_DIR)/$1.patch-applied: | $$(LLVM_SRC_DIR).extracted $$(SRCDIR)/patches/$1.patch $$(LLVM_PATCH_PREV)
+$$(LLVM_SRC_DIR)/$1.patch-applied: $$(LLVM_SRC_DIR)/source-extracted | $$(SRCDIR)/patches/$1.patch $$(LLVM_PATCH_PREV)
 	cd $$(LLVM_SRC_DIR) && patch -p1 < $$(SRCDIR)/patches/$1.patch
 	echo 1 > $$@
 LLVM_PATCH_PREV := $$(LLVM_SRC_DIR)/$1.patch-applied
-LLVM_PATCH_LIST += $$(LLVM_PATCH_PREV)
```

Comment:
I think removing this would mean that if anyone adds a patch later on and it's not at the very end of the list, it would get skipped